apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +253,78 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+}: {
+  csrfToken: string;
+  initialStartDateFormatted?: string;
+  initialEndDateFormatted?: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create Course Instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="short_name">CIID (Course Instance Identifier)</label>
+        <input
+          type="text"
+          class="form-control"
+          id="short_name"
+          name="short_name"
+          required
+          pattern="[\\-A-Za-z0-9_\\/]+"
+          title="Please enter a valid format: only letters, numbers, hyphens, underscores, backslashes, and forward slashes."
```

Comment:
`title` is generally not considered very accessible: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns. I think we can just drop this; the label below the form should be sufficient.